src/main_controller.ts

```diff
@@ -184,8 +186,21 @@ export class MainController implements vscode.Disposable {
             }
         }
 
+        let neovimPath = config.neovimPath;
+        // Only try to find nvim if the path is the default one
+        // And if we are not using WSL
```

Comment:
could findNvim() be enhanced to work with WSL? PR welcome at https://github.com/neovim/node-client/

Reply:
I personally think it's not very meaningful. The use case for nvim in WSL is limited, and WSL essentially functions like a virtual machine.